packages/atomic/cypress/integration/facets/color-facet/color-facet.cypress.ts

```diff
@@ -233,7 +233,13 @@ describe('Color Facet Test Suites', () => {
     describe('repeatedly until there\'s no more "Show more" button', () => {
       function setupRepeatShowMore() {
         new TestFixture()
-          .with(addColorFacet({field: colorFacetField, label: colorFacetLabel}))
+          .with(
+            addColorFacet({
+              field: colorFacetField,
+              label: colorFacetLabel,
+              'number-of-values': 25,
```

Comment:
I did not see an error with the color facet, but I applied the same idea. The field has 46 values, so 7 "show more" clicks are needed by default. If the initial number is 25, only one click is needed.